app/controllers/api/memberships_controller.rb

```diff
@@ -1,5 +1,16 @@
 class API::MembershipsController < API::RestfulController
 
+  def add_to_subgroup
+    parent_group = current_user.groups.find_by_id!(params[:parent_group_id])
+    group = current_user.groups.find_by_id!(params[:group_id])
+
+    users = parent_group.members.where('users.id IN (?)', params[:user_ids])
```

Comment:
Could be `parent_group.members.where('users.id': params[:user_ids])` for slightly less SQL (this makes it so it doesn't error when params[:user_ids] is blank)